README.md

```diff
@@ -114,7 +114,7 @@ A collection of awesome things regarding React ecosystem.
 * [React Core meeting notes](https://github.com/reactjs/core-notes)
 
 #### React Online Playground
-* [React JSFiddle Integration with JSX](https://jsfiddle.net/reactjs/69z2wepo/)
+* [React JSFiddle Integration with JSX](https://jsfiddle.net/quadroid/a5hd28s6/)
```

Comment:
Instead, probably worth sending to the React team to get them to fix their version and set as base :rose: